pkg/helm/testdata/bad-install-input.missing-desc.yaml

```diff
@@ -0,0 +1,5 @@
+install:
+- helm:
+    name: wordpress
+    chart: stable/wordpress
+    description: ""
```

Comment:
I have a validation on description that ensures that it is at least 1 char long.